src/blueprints/tv2/factories/tv2-base-graphic-timeline-object-factory.ts

```diff
@@ -0,0 +1,18 @@
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { Tv2GraphicsLayer } from '../value-objects/tv2-layers'
+import { Tv2GraphicsTarget } from '../value-objects/tv2-graphics-target'
+import { Tv2GraphicActionManifest } from '../value-objects/tv2-action-manifest'
+
+export abstract class Tv2BaseGraphicTimelineObjectFactory {
```

Comment:
This is only used by one class. Don't create abstraction before you need them.
Furthermore: This class only have one method and the child class has an almost identical method that switch on the excat same value. Why is that also not moved here?

This shouldn't be created untill you actually have a need for it, which is not yet.

Reply:
Is only used by one _commited_ class :P
I haven't reached the point where i got the full overview of what overlabs between the Viz and CasparCG factories, but i'm fairly certain there is going to be a couble more methods atleast.